### PR TITLE
UI improvement

### PR DESCRIPTION
**Pull request by lvenier**
backporting some of our internal agent UI feature into this version

### CODE REVIEW

File: flows.json [hunk 1]
```diff
@@ -1232,6 +1232,29 @@
             []
         ]
     },
+    {
+        "id": "4b6dba9f0b7031fc",
```
Comment (fanshan) [inline flows.json:1236]: image
 Not needed I think. This test is done before the read file.

**Review by fanshan — CHANGES_REQUESTED** [review]

File: flows.json (hunk 1 above)
Comment (lvenier) [reply flows.json:1236]: needed when we delete image because we go to the init

Comment (fanshan) [reply flows.json:1236]: I don't see the relation between delete a image and connection with this catch node.

Comment (lvenier) [reply flows.json]: fixed. Was for the journal.

**Review by fanshan — APPROVED** [review]

**Pull request merged**